### PR TITLE
update some hover styles for better look

### PR DESCRIPTION
**Pull request by akshayvinchurkar**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -868,15 +868,22 @@ a:visited .nav-title,
...
 .widget_authors a:hover strong,
 .project-terms a:focus,
 .project-terms a:hover {
 	border-bottom-color: transparent;
 	background-color: #767676;
 	color: #fff;
+	transition: all 0.3s linear;
+}
+
+.comment-reply-link:hover{
```
Comment (ntwb) [inline style.css]: Expected a single space before the open `{` brace

i.e. `.comment-reply-link:hover {`, not `.comment-reply-link:hover{`

File: style.css [hunk 2]
```diff
@@ -868,15 +868,22 @@ a:visited .nav-title,
...
+	transition: all 0.2s linear;
+	background-color: #767676;
+	color: #fff;
+	padding: 2px 5px 2px 5px;
```
Comment (ntwb) [inline style.css]: Use shorthand `padding: 2px 5px;` rather than `padding: 2px 5px 2px 5px;`

**Review by ntwb — CHANGES_REQUESTED** [review]
Add some coding standards issues to be fixed.

**Review by akshayvinchurkar — COMMENTED** [review]
Please re-review the changes and merge the request

**Comment by melchoyce** [comment]
Can you provide before and after screenshots of what you're changing in this PR?

File: style.css [hunk 3]
```diff
@@ -3338,4 +3345,4 @@ article.panel-placeholder {
 	.twentyseventeen-front-page .entry-content blockquote.alignright {
 		margin-right: -20%;
 	}
-}
\ No newline at end of file
+}
```
Comment (ntwb) [inline style.css:3348]: Don't delete the newline at the end of the file.

**Review by ntwb — CHANGES_REQUESTED** [review]

File: style.css (hunk 2 above)
Comment (josephfusco) [inline style.css]: We shouldn't be changing padding on hover as it affects layout and does not offer a high performance transition.

See: [https://www.html5rocks.com/en/tutorials/speed/high-performance-animations/#toc-animating-layout-properties](https://www.html5rocks.com/en/tutorials/speed/high-performance-animations/#toc-animating-layout-properties)

**Comment by josephfusco** [comment]
@melchoyce https://cloudup.com/cYlHVZQec08

**Comment by melchoyce** [comment]
Thanks for the video, @josephfusco. @akshayvinchurkar, thanks for your contribution. Looking at the change, I don't think this is a good fit for Twenty Seventeen. I'm going to close this PR, but thanks for your thoughts and for experimenting.

**Comment by akshayvinchurkar** [comment]
on hover it looks like he cant breeth in that small area 😅

**Comment by melchoyce** [comment]
We're look at overall link hovers in #222 and #240. Please keep an eye on that ticket for updates :)

**Pull request closed without merging**